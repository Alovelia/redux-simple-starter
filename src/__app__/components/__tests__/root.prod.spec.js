import { shallow } from 'enzyme';
import React from 'react';
import { createMockStore } from 'redux-test-utils';
import Root from '../root.prod';

describe('<Root />', () => {
  const props = {
    history: {},
    rootRoute: [],
    store: createMockStore()
  };
  test('when initializing, the expected output is rendered', () => {
    expect(shallow(<Root {...props} />)).toMatchSnapshot();
  });
});
