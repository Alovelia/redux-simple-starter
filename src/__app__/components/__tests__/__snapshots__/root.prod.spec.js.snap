// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Root /> when initializing, the expected output is rendered 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Root
    history={Object {}}
    rootRoute={Array []}
    store={
      Object {
        "dispatch": [Function],
        "getAction": [Function],
        "getActions": [Function],
        "getState": [Function],
        "isActionDispatched": [Function],
        "isActionTypeDispatched": [Function],
        "subscribe": [Function],
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "class",
    "props": Object {
      "children": <Connect(I18nContainer)>
        <Router
          history={Object {}}
          render={[Function]}
          routes={Array []}
        />
      </Connect(I18nContainer)>,
      "store": Object {
        "dispatch": [Function],
        "getAction": [Function],
        "getActions": [Function],
        "getState": [Function],
        "isActionDispatched": [Function],
        "isActionTypeDispatched": [Function],
        "subscribe": [Function],
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "children": <Router
          history={Object {}}
          render={[Function]}
          routes={Array []}
        />,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "history": Object {},
          "render": [Function],
          "routes": Array [],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "children": <Connect(I18nContainer)>
          <Router
            history={Object {}}
            render={[Function]}
            routes={Array []}
          />
        </Connect(I18nContainer)>,
        "store": Object {
          "dispatch": [Function],
          "getAction": [Function],
          "getActions": [Function],
          "getState": [Function],
          "isActionDispatched": [Function],
          "isActionTypeDispatched": [Function],
          "subscribe": [Function],
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "children": <Router
            history={Object {}}
            render={[Function]}
            routes={Array []}
          />,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "history": Object {},
            "render": [Function],
            "routes": Array [],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
